projects/kit/components/avatar/avatar.module.ts

```diff
@@ -1,9 +1,10 @@
 import {CommonModule} from '@angular/common';
 import {NgModule} from '@angular/core';
+import {TuiLazyLoadingModule} from '@taiga-ui/kit/directives/lazy-loading';
```

Comment:
@MarsiBarsi should we import from directives entry point? Maybe we should document it somehow?

Reply:
We can but we should not. It is not very important for me, but I see a small benefit here: it may be easier for IDE to add the next import from directives. Well, we can describe such "import policy" in our CONTRIBUTING.md and I am going to fix it here with web editor of Github